src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -42,14 +42,29 @@ const NotebookLauncher = _.flow(
     showTabBar: false
   })
 )(
-  ({ queryParams, notebookName, workspace, workspace: { workspace: { namespace, name }, accessLevel, canCompute }, runtimes, persistentDisks, refreshRuntimes },
+  ({ queryParams, notebookName, workspace, workspace: { workspace: { bucketName, googleProject, namespace, name }, accessLevel, canCompute }, runtimes, persistentDisks, refreshRuntimes },
     ref) => {
     const [createOpen, setCreateOpen] = useState(false)
     const runtime = getCurrentRuntime(runtimes)
     const { runtimeName, labels } = runtime || {}
     const status = getConvertedRuntimeStatus(runtime)
     const [busy, setBusy] = useState()
     const { mode } = queryParams
+    const [location, setLocation] = useState(defaultLocation)
```

Comment:
We usually cluster together `state`s. Not sure why we didn't in this file but it'd be nice to start:

```suggestion
    const [createOpen, setCreateOpen] = useState(false)
    const [busy, setBusy] = useState()
    const [location, setLocation] = useState(defaultLocation)

    const runtime = getCurrentRuntime(runtimes)
    const { runtimeName, labels } = runtime || {}
    const status = getConvertedRuntimeStatus(runtime)
    const { mode } = queryParams

```

Reply:
Done.